.github/workflows/ci.yml

```diff
@@ -61,6 +61,14 @@ jobs:
         with:
           java-version: temurin@17
 
+      - name: Get mill version
+        run: echo "MILL_VERSION=$(cat .mill-version)" >> $GITHUB_ENV
+
+      - name: Setup Mill
```

Comment:
we don't need this if we're not using integration tests, do we ?

Reply:
that's right I forgot to remove it